README.md

```diff
@@ -50,31 +50,6 @@ For Linux users, revert to the instructions for installing phpcs, phpstan and pr
 * <https://phpstan.org/user-guide/getting-started>
 * <https://pre-commit.com/index.html#install>
 
-### Running locally without 3rd party integrations
```

Comment:
How come we are removing this whole part of the readme? I thought they would still be relevant

Reply:
Running without integrations does not work at present, because you have to have a Notify API key which is valid. The aim is to remove that dependency as we discussed this morning, so you could then run the test suite with no deps on external API keys etc.